openslides/users/static/js/users/users.js

```diff
@@ -187,30 +201,73 @@ angular.module('OpenSlidesApp.users', [])
     };
 }])
 
-.controller('UserListCtrl', function($scope, User, i18n) {
+.controller('UserListCtrl', function($scope, User) {
     User.bindAll({}, $scope, 'users');
+    $scope.sortby = 'first_name';
+    $scope.reverse = false;
+    $scope.filterPresent = '';
+
+    $scope.togglePresent = function (user) {
+        User.save(user);
+    };
```

Comment:
Are you sure that this work? You can create a TODO or a issue if you want to do it later